site/en/docs/extensions/reference/userscripts/index.md

```diff
@@ -0,0 +1,60 @@
+---
+api: userScripts
```

Comment:
Could we add a note somewhere that user scripts are cleared away on extension update, so you should listen to runtime.onInstalled with the update reason and add them back?

Reply:
Where should they do this? The service worker?

Reply:
Yeah.